### ISSUE TITLE
[New Notebook]: Something with the National Maine Sanctuaries and IOOS data services

### ISSUE THREAD

**Issue by MathewBiddle** [ioos/ioos_code_lab#170]
### Programming Language

Python

### Module/Software/IOOS data service?

IOOS ERDDAP, arcGIS feature service

### Module/Software/IOOS data service?

https://github.com/MathewBiddle/sandbox/blob/main/notebooks/union_RA_boundary_and_sanctuaries.ipynb

### Provide a description.

I keep hearing about collaborations with IOOS and the National Marine Sanctuaries Office and I developed this notebook to find the intersection of IOOS RA boundaries and Sanctuaries boundaries. https://github.com/MathewBiddle/sandbox/blob/main/notebooks/union_RA_boundary_and_sanctuaries.ipynb Is there something we can do with this to add to the Code Lab? I'm drawing a blank on highlighting IOOS services with this effort, but I feel it should be in the code lab in some way

**Comment by MathewBiddle** [comment]
@ocefpaf mentioned this could be defining a ROI and finding IOOS data within it. 

It's a bummer the ERDDAP API can't subset by wkt polygons. https://github.com/ERDDAP/erddap/issues/55

extractr has been able to do some fancy ERDDAP ROI queries: https://marinebon.github.io/extractr/articles/extractr.html But I think it's pulling a bounding box for the ROI, then clipping that bounding box to the ROI...

**Comment by ocefpaf** [comment]
> It's a bummer the ERDDAP API can't subset by wkt polygons. [ERDDAP/erddap#55](https://github.com/ERDDAP/erddap/issues/55)

I don't think we will be able to do that on the server side due to the 4096 characters per request URL limitation in Tomcat. Most ROI would be bigger than that to be precise.

> extractr has been able to do some fancy ERDDAP ROI queries: https://marinebon.github.io/extractr/articles/extractr.html But I think it's pulling a bounding box for the ROI, then clipping that bounding box to the ROI...

That is probably the best approach we have for now. We do the same in erddapy. See https://ioos.github.io/erddapy/01a-griddap-output.html for an example.

**Comment by MathewBiddle** [comment]
query obis for these regions. percent cover for mussels changing over time.

**Comment by MathewBiddle** [comment]
cc @laurabrenskelle

**Comment by MathewBiddle** [comment]
These data https://obis.org/dataset/fbf4cb59-cef3-43be-b0ab-0dbab5f51740. Which should be updated with additional data.

cc @sformel-usgs
